src/components/Header.tsx

```diff
@@ -17,8 +16,8 @@ export const HEADER_HEIGHT = Platform.select({
 }) as number;
 
 interface Props {
-  navigation?: StackNavigationProp<any, any>;
-  title: string;
+  navigation?: any;
```

Comment:
I spent an hour trying to fix that but without results :( Fixing it required some experience with React Native, I think

Reply:
can we //ts-ignore all the places with wrong navigation? I would like to have still typescript watching the navigation props

Reply:
Can be done it requires adding `// @ts-ignore` in 25 places.

Reply:
I would do it anyway, benefit from it is that we will have still navigation validation for new screens and that are currently correct

Reply:
Done.